README.md

```diff
@@ -8,25 +8,25 @@
 * **design your OpenStack product?**
   You would be primarily interested by [the OpenStack requirement list](doc/ref_arch/openstack/chapters/chapter02.md)
   and [OpenStack API descriptions and selection](doc/ref_arch/openstack/chapters/chapter05.md).
-* **check the CNTT conformance of your existing OpenStack deployments?**
+* **check the Anuket conformance of your existing OpenStack deployments?**
   You would be primarily interested by
-  [the NFVI Testing Cookbook](doc/ref_cert/RC1/chapters/chapter04.md)
+  [the Cloud Infrastructure Testing Cookbook](doc/ref_cert/RC1/chapters/chapter04.md)
```

Comment:
Until we change the title of the chapter, we cannot make this change here -- the title of the chapter is "NFVI Testing Cookbook"

Reply:
Oh, really. We should change the title then :)

Reply:
Related pr: #2532

Reply:
IN RC2, the chapter title is "Chapter 03 - Kubernetes Testing Cookbook", so it would seem like the equivalent here would be "OpenStack Testing Cookbook."  Otherwise, I think NFVI at least implies the VM aspects of this work, while "cloud" might start the blur the lines a little bit.

Reply:
Fixed in 9fbd62c

Reply:
@lylavoie @CsatariGergely but it is not an OpenStack Testing Cookbook. Why not change the sentence to "check the Anuket conformance of your existing OpenStack deployments? You would be primarily interested in the Testing Cookbook (with link to the chapter) for OpenStack based Cloud Infrastructure and the test case descriptions and selection for more details."

The RC1 chapter Title can also be changed

Reply:
@pgoyal01 What would you propose for the chapter titles? I'm good with your other proposed changes, i.e. "existing k8s / OpenStack" for the descriptive text.  For the chapter titles, I just want some clear consistency between RC1 and RC2, and avoid ambiguous terms like "Cloud," that doesn't track back to specific technologies.

Reply:
@lylavoie would adding "-based Infrastructure" be acceptable, as in, "Kubernetes-based Infrastructure Testing Cookbook" and "Kubernetes-based Infrastructure Testing Cookbook"?

Reply:
@pgoyal01  works for me, as long as one of those is "OpenStack-based".  ;)

Reply:
@lylavoie Sorry -- haste with cut-n-paste. Yes OpenStack Chapter 3 would have to be "OpenStack-based Infrastructure Testing Cookbook"

Reply:
@pgoyal01 yeah, I was just giving you a "hard-time."  I'm good with that approach, since it's reasonably descriptive.  Are you going to make an update the branch?

Reply:
@CsatariGergely Can you make the changes in this branch to RC1 and RC2 Chapter 03 titles? Thanks